appj.adoc

```diff
@@ -25,52 +25,105 @@ When an interpolation method is referred to as linear or quadratic, it means tha
 
 For convenience, an interpolation variable `s` is introduced, calculated as a function of the index in the target domain of the coordinate value to be reconstituted. In the case of one dimensional interpolation the interpolation variable is computed as 
 
-`s = (i - ia)/(ib - ia)`
+`s = s(i) = (i - ia)/(ib - ia)`
 
 where `ia` and `ib` are the indices in the target domain of the tie points `A` and `B` and `i` is the index in the target domain of the coordinate value to be reconstituted. 
 
 Note that the value of `s` varies from `0.0` at the tie point `A` to `1.0` at tie point `B`. For example, if `ia = 100` and `ib = 110` and the index in the target domain of the coordinate value to be reconstituted is `i = 105`, then `s = (105 - 100)/(110 - 100) = 0.5`.
 
 In the case of two dimensional interpolation, the two interpolation variables are equivalently computed as
 
-`s1 = (i1 - ia1)/(ib1 - ia1)`  +
-`s2 = (i2 - ia2)/(id2 - ia2)`
+`s1 = s1(i) = (i1 - ia1)/(ib1 - ia1)`  +
+`s2 = s2(i) = (i2 - ia2)/(id2 - ia2)`
 
-where `ia1` and `ib1` are the first dimension indices in the target domain of the tie points `A` and `B`, `ia2` and `id2` are the second dimension indices in the target domain of the tie points `A` and `D` and the indices `i1` and `i2` are the first and second dimension indices in the target domain of the coordinate value to be reconstituted. 
+where `ia1` and `ib1` are the first dimension indices in the target domain of the tie points `A` and `B` respectively, `ia2` and `id2` are the second dimension indices in the target domain of the tie points `A` and `D respectively and the indices `i1` and `i2` are the first and second dimension indices respectively in the target domain of the coordinate value to be reconstituted. 
 
 For the reconstitution of the uncompressed coordinate and auxiliary coordinate variables the interpolation method can be applied independently for each interpolation zone, making it possible to parallelize the computational process.
 
+The following notation is used: +  
+For a vector `v`,  `v.x` , `x.y` and `v.z` refer to the three coordinates of that vector. +
+For a coordinate pair `ll`, `ll.lat` and `ll.lon` refer to the latitude and longitude coordinates. +
 
+[[coordinate_conversion]]
+==== Common conversions and formulas
+
+[cols="8,12"] 
+|=============== 
+| Description | Formula
```

Comment:
I think adding a column for function names (fll2xyz, fcross, norm, etc...) would make it easier for users to find function descriptions when they encounter the function names elsewhere in the document.

Reply:
Could we achieve that by adding the function name in bold as the first line in existing Description column?
Like in 

**fll2xyz()**
Conversion from geocentric (latitude, longitude) to cartesian vector (x, y, z)